### DISCUSSION TITLE
bticino x8000: non si aggiorna automaticamente il sensore di temperatura/umidità

### DISCUSSION THREAD

**Discussion by Vinz1985** [indomus/forum#449] [HUB: Home Assistant]
Salve, ho integrato tempo fa il termostato della ticino seguendo la guida e tutto funziona, però mi sono accorto di una cosa.: noto che la temperatura che leggo sulla card non si aggiorna mai in automatico al variare delle condizioni ambientali, (sebbene sia sul termostato sia sull app si aggiorna al variare di tali condizioni); il valore del sensore si aggiorna solo se effettuo un operazione  (cambio temperatura, imposto il boost, imposto un programma, ecc...). 
Questo non aggiornamento del sensore della temperatura in automatico non mi permette di gestire, tramite l'apposito sensore varie automazioni (accensione/spegnimento condizionatore, accensione/spegnimento termosifoni).  
C'è un modo per forzare tale aggiornamento? 
Eppure nel log non mi da nessun errore, è strano.

**Comment by Matmos90** [comment]
Ciao anche io ho il tuo stesso problema come hai risolto?

**Reply by Moxser** [reply]
Probabilmente il problema che avete è legato al limite di chiamate che l'add on può fare al giorno. Se non sbaglio dovrebbero essere 500. Per questo non ho mai integrato i 5 termostati della btcino che ho installato anni fa.

**Reply by DjMayone** [reply]
Ciao a tutti, anche io ho lo stesso problema. Avete poi risolto? Sono quasi sul punto di sostituire il vecchio x8000 con x8000v2 compatibile con l’app Netatmo. Mi dispiacerebbe però…

**Reply by Matmos90** [reply]
Ciao si ho risolto facendo in questo modo:
binary_sensor:
- platform: rest
  resource: http://IndirizzoIPMqttBroker:5588/info
  name: updatetempurature
  scan_interval: 180
Io avevo il problema che la temperatura non mi si aggiornava fino a che non aprivo l interfaccia web del componente aggiuntivo.
Lo scan interval non puoi scendere sotto i 180 secondi perché altrimenti supereresti il limite di chiamate giornalieri.

**Reply by DjMayone** [reply]
Ciao @Matmos90 grazie mille per la info! Ultima cosa: dove hai inserito queste correzioni? 
Grazie ancora!

**Reply by Matmos90** [reply]
Puoi inserire il tutto all'interno del tuo file configuration.yaml.

**Reply by DjMayone** [reply]
Copio così com’è aggiornando solo ip del mio broker giusto? 
ultima cosa, nel caso avessi due x8000 compilo una volta sola giusto?

**Reply by Matmos90** [reply]
Risposta1: esatto, volendo alla riga name: updatetempurature puoi dare il nome che vuoi basta che non contanga spazi oppure lettere maiuscole.
Risposta2: penso di si ma non saprei in quanto io ne ho uno solo.

**Reply by DjMayone** [reply]
Eccomi qui ancora! Ho applicato la modifica al configuration.yaml come mi hai indicato ma, al momento di verificare la configurazione su HA, mi vengono restituiti questi errori. hai idea di cosa possa essere?

Invalid config for [binary_sensor.rest]: must contain at least one of resource, resource_template.. Got {'platform': 'rest'}. (See ?, line ?). 
Invalid config for [binary_sensor]: required key not provided @ data['platform']. Got None. (See /config/configuration.yaml, line 25). 
Invalid config for [binary_sensor]: required key not provided @ data['platform']. Got None. (See /config/configuration.yaml, line 25). 
Invalid config for [binary_sensor]: required key not provided @ data['platform']. Got None. (See /config/configuration.yaml, line 25).

**Reply by Matmos90** [reply]
![2023-07-20 14_13_54-_nuovo 1 - Notepad++](https://github.com/indomus/forum/assets/119707291/7467711e-45af-43f2-b970-3387fe9ac80a)

**Reply by DjMayone** [reply]
Perfetto! Ora il file yaml non mi restituisce errori.
Grazie infinite ancora!
